frontend/src/scenes/dashboard/DashboardHeader.tsx

```diff
@@ -264,17 +264,17 @@ export function DashboardHeader(): JSX.Element {
                                 allowClear
                             />
                         ) : (
-                            dashboard.description || (
-                                <Button
-                                    type="link"
-                                    onClick={() =>
-                                        setDashboardMode(DashboardMode.Edit, DashboardEventSource.AddDescription)
-                                    }
-                                    style={{ width: '100%', textAlign: 'left' }}
-                                >
-                                    Add a description...
-                                </Button>
-                            )
+                            <div
+                                onDoubleClick={() =>
+                                    setDashboardMode(DashboardMode.Edit, DashboardEventSource.AddDescription)
+                                }
+                            >
+                                {dashboard.description ? (
+                                    <span>{dashboard.description}</span>
+                                ) : (
+                                    <span className="add-description">Add a description...</span>
+                                )}
+                            </div>
```

Comment:
It's not super clear that this can be simply clicked. My suggestion: show icon `<EditOutlined/>` on the right when not in edit mode.

Reply:
<img width="1060" alt="Screen Shot 2021-04-23 at 10 51 35 AM" src="https://user-images.githubusercontent.com/25164963/115889582-11d5ce00-a422-11eb-8dea-7b71c71d6604.png">